config/initializers/devise.rb

```diff
@@ -36,7 +36,7 @@
   #
   # Uses the DevAuth strategy if local and ENV["DEV_AUTH_ENABLED"] is true
 
-  if !HostEnv.local? && FeatureFlags.dev_auth.enabled?
+  if HostEnv.production? && FeatureFlags.dev_auth.enabled?
     raise "The DevAuth strategy must not be used in this environment"
```

Comment:
Technically this error message is correct but is it worth being specific? - "The DevAuth strategy must not be used in production"